From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 18 Feb 2016 22:03:03 +0000
Subject: perf tools: Fix bpf feature check

Check that the bpf() system call is actually wired up for the target
architecture using #ifdef __NR_bpf.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/tools/build/feature/test-bpf.c
+++ b/tools/build/feature/test-bpf.c
@@ -1,4 +1,9 @@
 #include <linux/bpf.h>
+#include <asm/unistd.h>
+
+#ifndef __NR_bpf
+#error "bpf system call not wired for this architecture"
+#endif
 
 int main(void)
 {
